src/_reducers/SigninReducers.js

```diff
@@ -7,7 +7,6 @@ import {
 } from '../_constants/ActionTypes';
 
 const initialState = fromJS({
-    token: '',
     email: '',
     password: '',
     language: 'EN',
```

Comment:
Language too should go.
Or may be we keep token and language, but they are the viewModel thingies, that we do not access anywhere else.